src/account/operations/output_claiming.rs

```diff
@@ -231,6 +231,24 @@ impl AccountHandle {
             let mut new_native_tokens = NativeTokensBuilder::new();
             // check native tokens
             for output_data in outputs {
+                if let Some(native_tokens) = output_data.output.native_tokens() {
+                    // Skip output if the max native tokens count would be exceeded
+                    if new_native_tokens.len() + native_tokens.len() > NativeTokens::COUNT_MAX.into() {
```

Comment:
Is this correct? Because at this point there could be same Token IDs in both sides and they would be counted twice so we could ignore an output that was actually fine.

Reply:
It's claiming anyway, not a very big deal if we are being overly cautious I guess

Reply:
Should I do the following? It's two additional clones
```Rust
                    // Clone to get the new native token count without actually modifying it
                    let mut native_tokens_count = new_native_tokens.clone();
                    native_tokens_count.add_native_tokens(native_tokens.clone())?;
                    // Skip output if the max native tokens count would be exceeded
                    if native_tokens_count.len() > NativeTokens::COUNT_MAX.into() {
```